homeassistant/components/tuya/light.py

```diff
@@ -268,6 +268,45 @@ class TuyaLightEntityDescription(LightEntityDescription):
             brightness=DPCode.BRIGHT_VALUE_2,
         ),
     ),
+    # Wake Up Light II
+    # Not documented
+    "hxd": (
```

Comment:
Let's make sure we add some icons where appropriate and set the entity categories as well.